docs/components/ColorSwatch/ColorSwatch.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { ColorSwatch } from './ColorSwatch';
+
+const meta: Meta<typeof ColorSwatch> = {
+  title: 'Documentation Components/ColorSwatch',
+  component: ColorSwatch,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof ColorSwatch>;
+
+export const Default: Story = {
+  render: () => (
+    <ColorSwatch
+      color="#007bff"
+      borderColor="#ced4da"
+      textBackgroundColor="#f8f9fa"
+      textColor="#212529"
+      name="Primary Default"
+    />
+  ),
+};
```

Comment:
Adding missing `ColorSwatch` story for doc components

<img width="1511" alt="Screenshot 2024-01-22 at 5 26 18 PM" src="https://github.com/MetaMask/design-tokens/assets/8112138/a4fd5512-1414-4534-b799-144332744f07">